packages/channels/src/base/api.ts

```diff
@@ -14,19 +12,6 @@ export class ChannelApi<TService extends ChannelService<any, any>> {
   makeUrl(callback: (scope: string) => Promise<string>) {
     this.urlCallback = callback
   }
-
-  protected async printWebhook(scope: string, name: string, path?: string) {
-    // TODO: remove this dependency on server env vars
-    if (yn(process.env.SPINNED)) {
-      const externalUrl = await this.urlCallback!(scope)
-
-      this.service.logger?.info(
-        `[${scope}] ${clc.bold(name.charAt(0).toUpperCase() + name.slice(1))}${
-          path ? ' ' + path : ''
-        } webhook ${clc.blackBright(`${externalUrl}/${name}${path ? `/${path}` : ''}`)}`
-      )
-    }
-  }
```

Comment:
I'm removing this from here. I will still work with old channels but not the new ones. I suggest we change it so botpress prints this instead like we talked about